tests/test_utils_errors.py

```diff
@@ -11,117 +11,117 @@
     RepositoryNotFoundError,
     RevisionNotFoundError,
 )
-from huggingface_hub.utils._errors import REPO_API_REGEX, hf_raise_for_status
+from huggingface_hub.utils._http import REPO_API_REGEX, X_REQUEST_ID, _format, hf_raise_for_status
```

Comment:
Should the tests be moved to https://github.com/huggingface/huggingface_hub/blob/main/tests/test_utils_http.py for consistency with the rest of the codebase?